src/igvfd/types/file_set.py

```diff
@@ -1144,6 +1144,8 @@ def summary(self, request, file_set_type, scope, selection_criteria, small_scale
         criteria = []
         criteria = criteria + selection_criteria
 
+        if scope == 'control':
```

Comment:
Let me know your opinion on this, I wasn't sure how to integrate control_type into the summary. and at least for the current control examples we have it would seem most other metadata to be irrelevant